src/ui/sass/modules/_Results.scss

```diff
@@ -92,8 +92,8 @@
     padding: $base-spacing / 2 $base-spacing;
     background-color: $results-filters-background;
     border-top: $results-border;
-    border-right: $border-legacy;
-    border-left: $border-legacy;
+    border-right: $border-default;
```

Comment:
lgtm! do we want to make this not affect AccordionResults (and only VerticalResults) just to be extra safe?

Reply:
Not sure! It seems like AccordionResults might want to be changes as well, but idk. @tmeyer2115 do you have thoughts on this?

Reply:
hi all! (was added as a reviewer so I'll chime in, but Tom please feel free to correct!) 

AccordionResults currently has a more specific styling so I don't think this will apply to the filters on AccordionResults (line 159 of this file).
`      
&.yxt-Accordion .yxt-Results-filters {
  border-left: none;
  border-right: none;
}`

That is probably intended, because we have an open Github Issue to from Rose to change that AccordionResults styling (line 159 of this file) to:

`
&.yxt-Accordion .yxt-Results-filters {
  border-right: $border-legacy;
  border-left: $border-legacy;
}`

Reply:
@nbramblett, since Rose and @alexisgrow have basically laid out what needs to be done for the AccordionResults styling, why don't we include that here too. Two birds with one stone.